.github/workflows/release.yml

```diff
@@ -32,16 +32,16 @@ jobs:
   release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         with:
           ref: 'main'
           fetch-depth: 0
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: 16.x
-      - uses: pnpm/action-setup@v2
+          node-version: 18.x
+      - uses: pnpm/action-setup@v3
         with:
-          version: 8.10.0
+          version: 9.0.6
```

Comment:
We could have dependabot do this job for us 😉

Reply:
Yeah, I'll add a ticket to look at that